app/src/main/res/values/strings.xml

```diff
@@ -18,8 +18,10 @@
 
     <string name="oscilloscope">Oscilloscope</string>
     <string name="multimeter">Multimeter</string>
-    <string name="logical_analyzer">Logic Analyzer</string>
-    <string name="wavegen">Wave Generator</string>
+
+    <string name="logical_analyzer">Logical Analyzer</string>
```

Comment:
Revert this back to `Logic Analyzer`. It was a spelling mistake earlier in `Logical Analyzer`